### PR TITLE
Update children

### PR DESCRIPTION
**Pull request by Hanyman8**
Update word children

### CODE REVIEW

**Review by hroncok — COMMENTED** [review]
Thanks, but it's there in multiple locations. Would you be able to got trough all the document and find all?

File: cs/openscad.md [hunk 1]
```diff
@@ -189,20 +189,22 @@ Příklad: Modul
 
 ![Ukázka](../images/openscad/ex7.png)
 
-child()
+children()
 -------
 
--   `child(0)` .. `child($children-1)`
+-   `children(0)` .. `child($children-1)`
```
Comment (hroncok) [inline cs/openscad.md]: Still one child left on this line

**Comment by Hanyman8** [comment]
I update it in this file, tried use this code and the picture is same

**Review by hroncok — APPROVED** [review]

File: cs/openscad.md (hunk 1 above)
Comment (Hanyman8) [reply cs/openscad.md]: And the second one child

Comment (Hanyman8) [reply cs/openscad.md]: Fixed

**Comment by hroncok** [comment]
Thank you!

**Pull request merged**